packages/jellyfish-api-core/src/category/icxorderbook.ts

```diff
@@ -81,6 +81,29 @@ export class ICXOrderBook {
     )
   }
 
+  /**
+   * Create and submits a DFC HTLC transaction
+   *
+   * @param {HTLC} htlc
+   * @param {string} [htlc.offerTx] Transaction Id of the offer transaction for which the HTLC is
+   * @param {BigNumber} [htlc.amount] Amount in HTLC
+   * @param {string} [htlc.hash] Hash of seed used for the hash lock part
+   * @param {number} [htlc.timeout] Timeout (absolute in blocks) for expiration of HTLC in DFI blocks
+   * @param {UTXO[]} utxos Specific utxos to spend
```

Comment:
```suggestion
   * @param {UTXO[]} [utxos] Specific utxos to spend
```
here you need specific `optional` as well